build/specifications/Squirrel.json

```diff
@@ -0,0 +1,175 @@
+{
+  "$schema": "https://raw.githubusercontent.com/nuke-build/nuke/master/source/Nuke.CodeGeneration/schema.json",
+  "license": [
+    "Copyright 2018 Maintainers and Contributors of NUKE.",
+    "Distributed under the MIT License.",
+    "https://github.com/nuke-build/tools/blob/master/LICENSE"
+  ],
+  "references": [
+    "https://github.com/Squirrel/Squirrel.Windows/blob/master/src/Update/Program.cs"
+  ],
+  "name": "Squirrel",
+  "officialUrl": "https://github.com/Squirrel/Squirrel.Windows",
+  "help": "Squirrel is both a set of tools and a library, to completely manage both installation and updating your Desktop Windows application, written in either C# or any other language (i.e., Squirrel can manage native C++ applications).",
+  "packageId": "Squirrel.Windows",
+  "packageExecutable": "Squirrel.exe",
+  "tasks": [
+    {
+      "settingsClass": {
+        "properties": [
+          {
+            "name": "Install",
+            "type": "string",
+            "assertion": "DirectoryOrNull",
+            "format": "--install={value}",
+            "help": "Install the app whose package is in the specified directory."
+          },
+          {
+            "name": "Uninstall",
+            "type": "bool",
+            "format": "--uninstall",
+            "help": "Uninstall the app the same dir as Update.exe."
+          },
+          {
+            "name": "Download",
+            "type": "string",
+            "format": "--download",
```

Comment:
Missing `{value}` in format string.

Reply:
Those don't take parameters, so I guess "properties" is the wrong category here.
Is there an action in another tool that I can take as example which adds a parameter to the tool, but doesn't take an argument itself?